.evergreen/setup-venv.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
-# Enters a virtual environment or creates one named 'venv'.
-#
-
-echo $VIRTUAL_ENV
-if [ "$VIRTUAL_ENV" != "$(pwd)/venv" ]; then
-    echo "VIRTUAL_ENV=${VIRTUAL_ENV}, updating to use 'venv'"
-
-    if [ -d venv ]; then
-        echo "venv found: activating"
-        if [ "Windows_NT" == "$OS" ]; then
-            . ./venv/Scripts/activate
-        else
-            . ./venv/bin/activate
-        fi
-    else
-        echo "venv not found: creating, installing, and activating"
-        python -m virtualenv venv
```

Comment:
this line is problematic on Linux since we use python3 in these environments. Windows uses python2, so on Windows it works

Reply:
The alternative to removing can be code like this:
![image](https://user-images.githubusercontent.com/44870738/100465234-ff0fd500-30df-11eb-9f41-88c611308de8.png)
but I think removing it is the best option

Reply:
NOTE: python3 also has slight renaming for `virtualenv` module => `venv`